SPECS/nlohmann-json/nlohmann-json.spec

```diff
@@ -12,6 +12,7 @@ Source0:        %{name}-%{version}.tar.gz
 %global debug_package %{nil}
 BuildRequires:  cmake
 BuildRequires:  gcc
+BuildRequires:  git
```

Comment:
Can this be in a `with_check` block?

Reply:
No, since we are using `%autosetup -S git`. I think git is needed.